app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimAmountPage.scala.html

```diff
@@ -122,7 +122,7 @@
                 value = Some("GBP"),
                 conditionalHtml = Some(gbpInputHtml)
             )),
```

Comment:
I dont think the comma is needed here too.

Reply:
removed now